contents/java/JUnit5.md

```diff
@@ -0,0 +1,137 @@
+<frontmatter>
+  title: Introduction to JUnit 5
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Testing in Java with JUnit 5
+
+Author(s): [Lin Si Jie](https://github.com/sijie123)
+
+## What is JUnit 5?
+JUnit is a open-sourced automated testing framework for Java. Using its vast API library, you can easily write tests to ensure correctness in your software engineering project. After setting up JUnit, running tests within popular IDEs like IntelliJ is as easy as clicking a button. JUnit 5 is the latest iteration of the JUnit project, supporting multiple new features in Java (such as Lambdas) and adopting a new Extensions-based model to make writing tests easier, cleaner, and more modular.
+
+## Why do automated testing?
+There are many reasons why automated testing is important for any software engineering project. Automated testing
```

Comment:
> Automated testing

I think you should start this on a new line as your bullet points use `Automated testing` as the starting phrase. It will be more convenient to read.